src/components/App/App.spec.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from './App';
+import Calculator from '../Calculator/Calculator';
+
+describe('App', () => {
+    let wrapper;
+    beforeEach(() => {
+        wrapper = shallow(<App />);
+    })
+
+    it ('should render correctly', () => expect(wrapper).toMatchSnapshot())
+    
+    it ('should render a div', () => {
+        expect(wrapper.find('div').length).toEqual(1);
+    });
+
+    it ('should contain a Calculator Element', () => {
+        expect(wrapper.containsMatchingElement(<Calculator />)).toEqual(true)
```

Comment:
Now that we've mocked at the top, we can simply check for the string of Calculator.
```suggestion
        expect(wrapper.find('Calculator').length).toBe(1)
```